--- a/config/Config-build.in
+++ b/config/Config-build.in
@@ -203,7 +203,7 @@
 	choice
 		prompt "User space Stack-Smashing Protection"
 		depends on USE_MUSL
-		default PKG_CC_STACKPROTECTOR_REGULAR
+		default PKG_CC_STACKPROTECTOR_STRONG
 		help
 		  Enable GCC Stack Smashing Protection (SSP) for userspace applications
 		config PKG_CC_STACKPROTECTOR_NONE
@@ -221,7 +221,7 @@
 
 	choice
 		prompt "Kernel space Stack-Smashing Protection"
-		default KERNEL_CC_STACKPROTECTOR_REGULAR
+		default KERNEL_CC_STACKPROTECTOR_STRONG
 		depends on USE_MUSL || !(x86_64 || i386)
 		help
 		  Enable GCC Stack-Smashing Protection (SSP) for the kernel
